testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -84,7 +88,7 @@ Map cookiecutterDjangoApp = [
     repoName: 'cookiecutter-django-app',
     targetBranch: "master",
     pythonVersion: '3.5',
-    cronValue: cronOffHoursBusinessWeekday,
+    cronValue: cronOffHoursBusinessWeekdayLahore,
```

Comment:
This repo has been archived since it was moved into edx-cookiecutters, it can be removed from the list.

Reply:
Np. I've removed this app from the list.

Reply:
It's already replaced with `edx-cookiecutters`, just had to rebase the PR.